.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+name: Keycloak Client Nightly Release
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
```

The other workflow also checks for the latest Java version at all times, rather than using the cache always. To be consistent we can also add it here.
```suggestion
          java-version: 17
          check-latest: true
```

Additionally, we might also want to cache the Maven artifacts to speed up this task, but I can understand that might potentially be undesirable for a release pipeline.
```suggestion
          java-version: 17
          check-latest: true
          cache: maven
```

---

This would probably work okay for nightly releases, but the cache configured this way uses a checksum of all pom files, which change for every release.

---

> The other workflow also checks for the latest Java version at all times, rather than using the cache always. To be consistent we can also add it here.

Not sure what the need of that even is, I think we're absolutely fine just using whatever is cached/available on runners, and this is just a way to slow things down without any added benefits

---

Then let's remove it in the main workflow as well